src/content/admin/previews.mdx

```diff
@@ -14,16 +14,16 @@ The Previews section of the Admin Dashboard provides administrators with a centr
 
 ## Preview Scope
 
-Preview environments can have one of two scopes:
+Preview Environments can have one of two scopes:
 
-- **Global Scope** (default): The preview environment is accessible to all members of your organization. These previews are visible to the entire team and can be managed by users with appropriate permissions.
-- **Personal Scope**: The preview environment is only accessible to the owner and anyone they explicitly share it with. These previews are indicated with a user icon next to the preview name in the preview list.
+- **Global Scope** (default): The Preview Environment is accessible to all members of your organization. These previews are visible to the entire team and can be managed by users with appropriate permissions.
+- **Personal Scope**: The Preview Environment is only accessible to the owner and anyone they explicitly share it with. These previews are indicated with a user icon next to the preview name in the preview list.
 
-Preview environments use global scope by default. The scope determines who can view and interact with the preview environment within your organization.
+Preview Environments use global scope by default. The scope determines who can view and interact with the Preview Environment within your organization.
```

Comment:
Nit: We might add a link to the [permissions page](https://www.okteto.com/docs/core/user-roles-and-permissions/#operations-within-previews), so users can check the permissions they have